suites/tentacle/rados/tier-2_rados_test-brownfield.yaml

```diff
@@ -0,0 +1,334 @@
+
+  - test:
+      name: "issue repro-obj snap and pool snap deletion"
+      module: test_pool_snap.py
+      desc: obj snap deletion when pool snapshot is deleted on unfixed build
+      polarion-id: CEPH-83602685
+      config:
+        issue_reproduction: true
+
+  - test:
+      name: "issue repro-osd recovery with allocator file corruption"
+      module: test_osd_crashes.py
+      desc: setup osd failure due to allocator file corruption
+      polarion-id: CEPH-83609785
+      config:
+        verify_allocator_corruption:
+           issue_reproduction: true
+        pool_config:
+           pool_name: osd-alloc-pool
+           pg_num: 1
+           pg_num_max: 1
+           pool_type: replicated
+      comments: "8.1 bug-2338097"
+
+  - test:
+      name: Upgrade cluster to 7.1z2 ceph version
+      desc: Upgrade cluster to 7.1z2 version
+      module: test_upgrade_warn.py
+      polarion-id: CEPH-83574934
+      config:
+        command: start
+        service: upgrade
+        base_cmd_args:
+          verbose: true
+        args:
+          rhcs-version: 7.1
+          release: z2
+      abort-on-fail: false
+
+  - test:
+      name: "issue repro-osd recovery with allocator file corruption"
+      module: test_osd_crashes.py
+      desc: OSD fails with ceph_assert post upgrade
+      polarion-id: CEPH-83609785
+      config:
+        verify_allocator_corruption:
+           check_assert: true
+        pool_config:
+           pool_name: osd-alloc-pool
+
+  - test:
+      name: Upgrade cluster to latest 8.x ceph version
+      desc: Upgrade cluster to latest version
+      module: test_upgrade_warn.py
+      polarion-id: CEPH-83574934
+      config:
+        command: start
+        service: upgrade
+        base_cmd_args:
+          verbose: true
+        args:
+          rhcs-version: 8.0
+          release: rc
+      abort-on-fail: true
+
+  - test:
+      abort-on-fail: false
+      desc: "cephfs basic operations"
+      module: cephfs_basic_tests.py
+      name: cephfs-basics
+      polarion-id: "CEPH-11293"
+
+  - test:
+      name: nfs-ganesha_with_cephfs
+      module: nfs-ganesha_basics.py
+      desc: Configure nfs-ganesha on nfs server,do mount on any client and do IOs
+      polarion-id: CEPH-83574439
+      abort-on-fail: false
+
+  - test:
+      name: Upgrade cluster to latest 9.x ceph version
+      desc: Upgrade cluster to latest version
+      module: test_upgrade_warn.py
+      polarion-id: CEPH-83574934
+      config:
+        command: start
+        service: upgrade
+        base_cmd_args:
+          verbose: true
+        verify_cluster_health: true
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
+      name: "verify fix-osd recovery with allocator file corruption"
+      module: test_osd_crashes.py
+      desc: failed osd recovery due to allocator file corruption during upgrade
+      polarion-id: CEPH-83609785
+      config:
+        verify_allocator_corruption:
+          verify_fix: true
```

Comment:
This test would need to run after 8.x upgrade because 8.x would already contain the fix.
I do still think that we should skip the upgrade to 8.x altogether in this test suite, reason being - if the fix stops working in 9.0 suddenly, then we would never no because the failed OSD has already been recovered after upgrade to 8.x
By the time the cluster is about to upgrade to 9.0, it would already be in HEALTH_OK state.